share/kinesis-to-share/README.md

```diff
@@ -0,0 +1,209 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# kinesis-to-share
+
+This example will demonstrate how to publish data from [Amazon Kinesis](https://aws.amazon.com/kinesis/) to a [Vendia Share Uni](https://vendia.net/docs/share/dev-and-use-unis).  The point is to illustrate that partners to a Uni can take advantage of existing services they may already use, like Amazon Kinesis, to share information with partners.  In our scenario, a **Consignee** is placing an order for goods that should be published to a Uni that is comprised of a **Consignee**, **Shipper**, and **Carrier**.  In our scenario, the **Consignee** uses an order system that integrates with Amazon Kinesis.
+
+We will deploy the example using the [Vendia Share Command Line Interface (CLI)](https://vendia.net/docs/share/cli) and the [AWS Serverless Application Model (SAM)](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html).  Serverless resources like a [Kinesis](https://aws.amazon.com/kinesis/) stream and [AWS Lambda](https://aws.amazon.com/lambda/) function will be deployed.  Data will be published to an Kinesis stream via [generator.py](./generator.py), a Python3 script.  This, in turn, will trigger a Lambda function to parse order data and publish it to our node in the Vendia Share Uni.
+
+![kinesis-to-share Architecture](img/kinesis-to-share.png)
+
+# Pre-requisites
+
+* [Python3](https://www.python.org/download)
+
+* [AWS Serverless Application Model CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+
+* [AWS CLI version 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+
+* [Docker](https://docs.docker.com/install/)
+
+* [Vendia Share CLI](https://vendia.net/docs/share/cli)
+
+## Installing Python3 Dependencies
```

Comment:
Should this be after the instructions to clone the repo and switch working directories?

Reply:
Good call - made the change.